tests/core/full_node/test_full_node.py

```diff
@@ -806,87 +795,90 @@ async def test_new_transaction_and_mempool(self, wallet_nodes, bt, self_hostname
         peer = await connect_and_get_peer(server_1, server_2, self_hostname)
         incoming_queue, node_id = await add_dummy_connection(server_1, self_hostname, 12312)
         fake_peer = server_1.all_connections[node_id]
-        # Mempool has capacity of 100, make 110 unspent coins that we can use
         puzzle_hashes = []
 
         # Makes a bunch of coins
-        for i in range(5):
-            conditions_dict: Dict = {ConditionOpcode.CREATE_COIN: []}
-            # This should fit in one transaction
-            for _ in range(100):
-                receiver_puzzlehash = wallet_receiver.get_new_puzzlehash()
-                puzzle_hashes.append(receiver_puzzlehash)
-                output = ConditionWithArgs(ConditionOpcode.CREATE_COIN, [receiver_puzzlehash, int_to_bytes(100000000)])
-
-                conditions_dict[ConditionOpcode.CREATE_COIN].append(output)
-
-            spend_bundle = wallet_a.generate_signed_transaction(
-                100,
-                puzzle_hashes[0],
-                get_future_reward_coins(blocks[1 + i])[0],
```

Comment:
running this test in different blocks isn't relevant I take it?

Reply:
Yeah that is not important for this test. The point of this section is to fill the mempool. I decreased the size of the mempool so now we don't need to many TXs.